infra/templates/backend-state-setup/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "name" {
+  type        = "string"
+  description = "(Required) Specifies the human consumable label for this resource."
+  default     = ""
+}
+
+variable "location" {
+  type        = "string"
+  description = " (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created."
```

Comment:
required is redundant as the TF will fail with a similar note if its not provided as an arg